### PR TITLE
✅Feat: Added homepage

### PR DESCRIPTION
**Pull request by anjali1102**
Added Homepage 
Created Folder structure

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,25 @@
 import "./App.css";
-import logo from "./logo.png";
+import Mockman from "mockman-js";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { HomePage } from "./pages/HomePage/HomePage";
+import { ProductPage } from "./pages/ProductPage/ProductPage";
+import { WishlistPage } from "./pages/WishlistPage/WishlistPage";
+import { CartPage } from "./pages/CartPage/CardPage";
+import { LoginPage } from "./pages/LoginPage/LoginPage";
+import { LogoutPage } from "./pages/LogoutPage/LogoutPage";
```
Comment (devdeadviz) [inline src/App.js:9]: You can use an `index.js` file for all your exports and then you can import all your components from a single file.

File: src/components/homepage/Footer.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  --h2-font-size: 1.25rem;
+  --normal-font-size: 0.938rem;
+  --small-font-size: 0.75rem;
+  --mb-1: 0.5rem;
+  --mb-2: 1rem;
+  --mb-3: 1.5rem;
+  --mb-4: 2rem;
+  --z-index: 10;
+  --z-fixed: 100;
+}
```
Comment (devdeadviz) [inline src/components/homepage/Footer.css:34]: You can add all your variables to the `index.css` and use it anywhere in your codebase.

File: src/components/homepage/Footer.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  flex-direction: column;
+}
+/* 
+@media screen and (min-width: 48rem) :root {
+  --big-font-size: 6rem;
+  --h1-font-size: 2.25rem;
+  --h2-font-size: 1.5rem;
+  --normal-font-size: 1rem;
+  --small-font-size: 0.813rem;
+} */
```
Comment (devdeadviz) [inline src/components/homepage/Footer.css:124]: Remove commented codes, if it's not required.

File: src/components/homepage/Footer.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/anjalii1102">
+              <i class="fa fa-twitter"></i>
+            </a>
+          </li>
+          <li>
+            <a href="https://linkedin.com/in/anjali1102">
+              <i class="fa fa-linkedin"></i>
```
Comment (devdeadviz) [inline src/components/homepage/Footer.js:22]: Use `className` keyword instead of `class`.

File: src/pages/HomePage/HomePage.css [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+}
+
+.home__title p {
+  margin: 0;
+  padding: 0;
+  margin-top: 1rem;
+  margin-bottom: 0;
+}
+
+.home__title span {
```
Comment (devdeadviz) [inline src/pages/HomePage/HomePage.css:153]: Use the same class naming convention everywhere.

**Review by devdeadviz — APPROVED** [review]
Added some required changes, have a look at them.

File: src/App.js [hunk 6]
```diff
@@ -1,8 +1,25 @@
 import "./App.css";
```
Comment (govindappaarun) [inline src/App.js:1]: you can remove this import

File: src/components/homepage/Footer.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./Footer.css";
+const Footer = () => {
+  return (
+    <div>
```
Comment (govindappaarun) [inline src/components/homepage/Footer.js:5]: footer is also a block level element, can you remove extra div

File: src/components/homepage/Footer.css [hunk 8]
```diff
@@ -0,0 +1,124 @@
+:root {
```
Comment (govindappaarun) [inline src/components/homepage/Footer.css:1]: suggestion: keep homePage components directly under components, not required to group them

**Review by govindappaarun — APPROVED** [review]
I will approve as scope is directory structure and home page

**Pull request merged**